prisma/seed.ts

```diff
@@ -23,6 +25,118 @@ async function seed() {
     },
   });
 
+  await prisma.record.createMany({
+    data: [
+      // A record
+      {
+        id: 1,
+        username: username,
```

Comment:
When the key/value are the same, list it once: `username,`

Reply:
I see. Thank you.